### PR TITLE
Be 7 create entity hibernate dependency

### PR DESCRIPTION
**Pull request by itDarashukD**

### CODE REVIEW

**Review by shpaser2 — CHANGES_REQUESTED** [review]
the meaningful commit message is preferred. Could you change the commit message to "a new plugin was added to pom"?

File: .gitignore [hunk 1]
```diff
@@ -27,66 +27,4 @@ hs_err_pid*
 .idea
 mvnw.cmd
 mvnw
-target
```
Comment (shpaser2) [inline .gitignore:30]: restore the full gitignore file and make a new additional commit

File: .gitignore [hunk 2]
```diff
@@ -27,66 +27,4 @@ hs_err_pid*
...
-nbbuild/
-dist/
-nbdist/
-nbactions.xml
-nb-configuration.xml
-
-##########################
-## OS X
-##########################
-.DS_Store
```
Comment (shpaser2) [inline .gitignore:92]: restore the full gitignore file and make a new additional commit

**Review by shpaser2 — DISMISSED** [review]
look at comments

File: .gitignore (hunk 2 above)
Comment (itDarashukD) [reply .gitignore]: thanks, I was repair gitinore, and commit it

**Review by AlKaliada — DISMISSED** [review]
I think we should name the entity like in our DB, and please fix annotations NOT NULL and others with the requirements in DB (for example name can't be null, but linkedin can and others)

**Pull request merged**